src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs

```diff
@@ -184,17 +184,17 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
 
         private async Task UninstallAsync(IRPackageViewModel package) {
             _coreShell.AssertIsOnMainThread();
-            if (_selectedTab == SelectedTab.InstalledPackages) {
-                IsLoading = true;
-            }
+            IsLoading = true;
```

Comment:
No, it affects only `installed` and `loaded` tabs - `available` tab isn't changed, only some of its items will change `IsInstalled` and `IsLoaded` values, but the list will be the same.
So yes, instead of 
```{csharp}
if (_selectedTab == SelectedTab.InstalledPackages) {
    IsLoading = true;
}
```
it should be 
```{csharp}
if (_selectedTab == SelectedTab.InstalledPackages || _selectedTab == SelectedTab.LoadedPackages) {
    IsLoading = true;
}
```